src/doc/en/installation/conda.rst

```diff
@@ -73,51 +57,36 @@ environment for Sage development.
 
 Here we assume that you are using a git checkout.
 
-- Optionally, set the build parallelism for the Sage library. Use
-  whatever the meaningful value for your machine is - no more than
-  the number of cores::
-
-    $ export SAGE_NUM_THREADS=24
```

Comment:
Instead of removing optional stuff entirely I recommend moving it down to "notes" section. It might be useful sometimes e.g. I only want to use 3 CPUs for compilation to avoid the computer getting too slow doing something else.